src/js/components/StationList/index.ts

```diff
@@ -0,0 +1,44 @@
+import Component from "@/core/component";
+import view from "./view";
+import { StationUpdate } from "@/components";
+import { stationService, uiService } from "@/service";
+import { stationStore } from "@/store";
+import { $, createElement, closest } from "@/utils/dom";
+
+class StationList extends Component {
+  stationUpdate = new StationUpdate();
```

Comment:
```suggestion
  private stationUpdate = new StationUpdate();
```
외부에서 사용되는 변수가 아니기 때문에 private으로 제한해주면 좋을 것 같아요!